bin/app.js

```diff
@@ -155,6 +156,8 @@ function App(config) {
       onerror: req.logger.error({ err }),
     })(err);
   });
+
+  app.use(cookieParser());
```

Comment:
As Express middlewares are run in the same order in which they are declared, this must be moved to the top, otherwise `user_language` will always find an undefined `req.cookies` object as `cookieParser` won't have run on the request.
https://stackoverflow.com/a/25210756/7001932
For this kind of generic middleware, it's a good idea to put them just after the creation of the Express instance.